core/templates/pages/learner-group-pages/create-group/add-syllabus-items.component.html

```diff
@@ -0,0 +1,206 @@
+<div class="oppia-add-learner-group-syllabus">
+  <div class="add-learner-group-syllabus-desc-text">
+    <span translate="I18N_ADD_SYLLABUS_DESCRIPTION_TEXT"></span>
+  </div>
+  <div class="added-syllabus-container">
+    <div class="added-syllabus-item-card" *ngFor="let storySummary of syllabusStorySummaries; trackBy: index">
+      <div class="syllabus-item-image" [ngStyle]="{'background-color': storySummary.getThumbnailBgColor()}">
+        <img class="thumbnail-image syllabus-item-image" [src]="getStoryThumbnailUrl(storySummary)">
+      </div>
+      <div class="syllabus-item-title">{{ storySummary.getTitle() }}</div>
+      <span class="remove-syllabus" (click)="removeStoryFromSyllabus(storySummary.getId())"></span>
+    </div>
+    <div class="added-syllabus-item-card" *ngFor="let subtopicSummary of syllabusSubtopicSummaries; trackBy: index">
+      <div class="syllabus-item-image" [ngStyle]="{'background-color': subtopicSummary.thumbnailBgColor}">
+        <img class="thumbnail-image syllabus-item-image" [src]="getSubtopicThumbnailUrl(subtopicSummary)">
+      </div>
+      <div class="syllabus-item-title">{{ subtopicSummary.subtopicTitle }}</div>
+      <span class="remove-syllabus" (click)="removeSubtopicFromSyllabus(subtopicSummary.subtopicPageId)"></span>
+    </div>
+    <div *ngIf="syllabusStorySummaries.length == 0 && syllabusSubtopicSummaries.length == 0" class="no-syllabus-items-text">
+      <span translate="I18N_LEARNER_GROUP_NO_ITEMS_ADDED"></span>
+    </div>
+  </div>
+  <div class="learner-group-syllabus-filters-container">
+    <div class="oppia-same-row-container oppia-search-box-container">
+      <form class="search-bar-float-left oppia-search-bar-form" role="search">
+        <div class="form-group">
+          <div class="input-group">
+            <input type="text"
+                   class="form-control oppia-search-bar-text-input protractor-test-search-input"
+                   [placeholder]="searchBarPlaceholder"
+                   [(ngModel)]="searchQuery"
+                   (ngModelChange)="onSearchQueryChangeExec()"
+                   [ngModelOptions]="{ updateOn: 'blur', standalone:'true' }"
+                   aria-label="Search bar"
+                   (keydown.enter)="$event.target.blur()"
+                   (input)="searchToBeExec($event)">
+          </div>
+        </div>
+      </form>
+    </div>
+
+    <div class="oppia-same-row-container">
+      <div [ngClass]="{'open' : activeMenuName === 'type', 'dropup' : enableDropup}" ngbDropdown class="search-bar-float-left oppia-search-bar-type-selector dropdown" autoClose="outside">
```

Comment:
I see this enableDropup here but I don't see the variable in the component file.

Reply:
yeah, actually this needs to be removed. Thanks!

Reply:
fixed this in #15840